account_balance.go

```diff
@@ -26,13 +26,10 @@ import (
 
 type AccountBalance struct {
 	Hbars Hbar
-
 	// Deprecated: Use `AccountBalance.Tokens` instead
-	Token map[TokenID]uint64
-
-	// Deprecated
+	Token  map[TokenID]uint64
 	Tokens TokenBalanceMap
-	// Deprecated
+	// Deprecated: use a mirror node query
```

Comment:
Might be good idea to remove `Deprecated` for fields that are queried from mirror node. They are not actually being deprecated from the sdks